helperFunctions/subscribeHelpers/SalesforceAgentAssignedHelpers/HandleEndChatCallback.ts

```diff
@@ -23,7 +25,11 @@ export class HandleEndChatCallback {
 		private technicalDifficultyMessage: string,
 	) {}
 
-	public handleEndChat = async () => {
+	public handleEndChat = async (closeMessage?: string, event?: EventName) => {
+		if (event) {
+			this.modify.getAnalytics().sendEvent(getEventData(this.data.room.id, event));
+		}
+
```

Comment:
I see handleEndChat is called with the event for a couple of cases. In we can't cover all the case, maybe we should keep it separate?

Reply:
Yup, I tried handling this issue https://github.com/WideChat/Apps.Salesforce.LiveAgent/pull/75#discussion_r982043352 but I didn't consider the error cases. I will revert it to separate event sending.